charybdefs/project.clj

```diff
@@ -0,0 +1,8 @@
+(defproject jepsen-charybdefs "0.1.0-SNAPSHOT"
+  :description "charybdefs wrapper for use in jepsen"
+  :url "https://github.com/jepsen.io/jepsen"
+  :license {:name "Eclipse Public License"
+            :url "http://www.eclipse.org/legal/epl-v10.html"}
+  :dependencies [[org.clojure/clojure "1.8.0"]
```

Comment:
I think you can leave off the Clojure dependency here (move it to a :dev profile), since this is gonna be a library, not a standalone app.